WeekFormulas.ts

```diff
@@ -181,6 +181,10 @@ function setWeekReqs(weekNum: number): void {
         .setValues(weekReqs);
 }
 
+function test(): void{
```

Comment:
Do we need to have this test method in master? Or is this just for testing it on your side?

Reply:
Oh I missed what you said in your initial comment; you're good here.